base/challenge-skeleton/healthcheck/exploit/doit.py

```diff
@@ -14,6 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pwn import *
+import pwnlib
+
+def handle_pow(r):
+    r.recvuntil('./pow.py solve ')
+    challenge = r.recvline().strip()
+    proc = pwnlib.tubes.process.process('bypass_pow.sh')
+    solution = proc.readall().strip()
+    r.sendline(solution)
+    r.recvuntil('Correct\n')
```

Comment:
If this fails, will it be obvious by the error message what happened?

Reply:
nope, but if this fails is probably an infrastructure problem